src/Podenv/Runtime.hs

```diff
@@ -178,8 +178,9 @@ executePodman ctx = do
     Running -> do
       pure $ podmanExecArgs ctx
     Stopped -> do
-      P.runProcess_ (podman ["rm", cname])
-      pure $ podmanRunArgs re ctx
+      let prep = if ctx ^. keep then "start" else "rm"
+      P.runProcess_ (podman [prep, cname])
+      pure $ podmanExecArgs ctx
```

Comment:
I think we need to use `podmanRunArgs` after deleting a container, otherwise exec would fail with `no such container`. Perhaps we should use a guard instead, e.g.:

```haskell
Stopped | ctx ^. keep -> do
    P.runProcess_ (podman ["start", cname])
    pure $ podmanExecArgs ctx
Stopped -> do
    P.runProcess_ (podman ["rm", cname])
    pure $ podmanRunArgs re ctx
```